commit 4cbdfa028752
Author: Manish Goregaokar <manishsmail@gmail.com>
Date:   Tue Aug 1 12:57:42 2017 -0500

    servo: Merge #17934 - Remove doc comment on statement (from Manishearth:doc-comment); r=canaltinova
    
    rustc warns about these now
    
    Source-Repo: https://github.com/servo/servo
    Source-Revision: b49311c65a083a9b8eb03741bf7ea980b64f1e03
    
    --HG--
    extra : subtree_source : https%3A//hg.mozilla.org/projects/converted-servo-linear
    extra : subtree_revision : 6b3c7d7f4ffeb9e271c52d418cda42cac5dd378d
---
 servo/components/style/values/specified/image.rs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git servo/components/style/values/specified/image.rs servo/components/style/values/specified/image.rs
index 37ed00ccf443..687c6f17d432 100644
--- servo/components/style/values/specified/image.rs
+++ servo/components/style/values/specified/image.rs
@@ -686,14 +686,14 @@ impl LineDirection {
         input.try(|i| {
             let to_ident = i.try(|i| i.expect_ident_matching("to"));
             match *compat_mode {
-                /// `to` keyword is mandatory in modern syntax.
+                // `to` keyword is mandatory in modern syntax.
                 CompatMode::Modern => to_ident?,
                 // Fall back to Modern compatibility mode in case there is a `to` keyword.
                 // According to Gecko, `-moz-linear-gradient(to ...)` should serialize like
                 // `linear-gradient(to ...)`.
                 CompatMode::Moz if to_ident.is_ok() => *compat_mode = CompatMode::Modern,
-                /// There is no `to` keyword in webkit prefixed syntax. If it's consumed,
-                /// parsing should throw an error.
+                // There is no `to` keyword in webkit prefixed syntax. If it's consumed,
+                // parsing should throw an error.
                 CompatMode::WebKit if to_ident.is_ok() => {
                     return Err(SelectorParseError::UnexpectedIdent("to".into()).into())
                 },
diff --git servo/components/style/properties/gecko.mako.rs servo/components/style/properties/gecko.mako.rs
index 37ed00ccf443..687c6f17d432 100644
--- servo/components/style/properties/gecko.mako.rs
+++ servo/components/style/properties/gecko.mako.rs
@@ -2081,7 +2081,7 @@ fn static_assert() {
             return (parent_size, parent_unconstrained_size)
         }
 
-        /// XXXManishearth this should also handle text zoom
+        // XXXManishearth this should also handle text zoom
         let min = Au(parent.gecko.mScriptMinSize);
 
         let scale = (parent.gecko.mScriptSizeMultiplier as f32).powi(delta as i32);
